api/src/main/java/org/semanticweb/owlapi/io/RDFLiteral.java

```diff
@@ -161,6 +161,16 @@ public boolean isPlainLiteral() {
         return OWL2Datatype.RDF_PLAIN_LITERAL.getIRI().equals(datatype);
     }
 
+
+    /**
+     *
+     * @return true if this node is an RDF 1.1 Simple Literal
```

Comment:
I don't think this matches the definition. The RDF-1.1 Concepts and Abstract Syntax paragraph in question has two parts that I think are distinct:

https://www.w3.org/TR/rdf11-concepts/#section-Graph-Literal

The first is:

```
Please note that concrete syntaxes MAY support simple literals consisting of only a lexical form without any datatype IRI or language tag. Simple literals are syntactic sugar for abstract syntax literals with the datatype IRI http://www.w3.org/2001/XMLSchema#string.
```

Then the paragraph continues, but I think it is not referring to "simple literal" for the second part, partly because "similiarly" isn't used to extend definitions, it is just the opposite of "conversely", which is a contrasting/similarity term:

```
Similarly, most concrete syntaxes represent language-tagged strings without the datatype IRI because it always equals http://www.w3.org/1999/02/22-rdf-syntax-ns#langString.
```

I don't mind having a common boolean method here that covers both for consistency, but I don't think we should reuse the informal "simple literal" term for its name.